governance/polls/templates/Atlas/Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY
+summary: $sentence_summary
+discussion_link: $discussion_link
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY
+
+The $poll_deployers have placed an [Atlas Edit Monthly Cycle Proposal](https://sky-atlas.powerhouse.io/#A.1.10.2_Atlas_Edit_Monthly_Cycle-430185a5-fa5d-4664-89cf-21c9e6cfc109|0db3326e) into the [voting system](https://vote.makerdao.com/polling). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59%7C0db30308) will be active for fourteen days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to the following Atlas Edit Weekly Cycle Proposal:
+
+- [Atlas Edit Pull Request]($Pull_Request_link)
+- [Proposal Discussion Thread]($discussion_link)
```

Comment:
```suggestion
- [Proposal Discussion Thread]($discussion_URL)
```

Reply:
This is slightly more correct, but I understand it might not be a good idea to have different designations for the same data ("discussion_link" in the YAML directives). If this consideration has any weight, the proposed "Pull_Request_URL" should probably be disregarded too.

Reply:
We think matching the YAML is useful because a single find-and-replace can be used to update both locations. Although there is no reason the entry in the YAML couldn't also be updated to URL as well. What do you think?